tools/build_langserver/langserver/analyzer.go

```diff
@@ -386,23 +482,33 @@ func (a *Analyzer) VariablesFromContent(content string, pos lsp.Position) map[st
 			continue
 		}
 
-		var varType string
-		if i.Type == "assign" {
-			varType = getVarType(i.Action.Assign.Val)
-		} else if i.Type == "augAssign" {
-			varType = getVarType(i.Action.AugAssign.Val)
+		variable := a.VariableFromIdent(i)
```

Comment:
nit: it's idiomatic to do this inline in the if
```suggestion
		if variable := a.VariableFromIdent(i); variable != nil {
```